docs/template-migration/guide-to-leveraging-ai.md

```diff
@@ -0,0 +1,22 @@
+# Guide to leveraging AI for template migration
+
+## Overview
+
+The general flow goes like this:
+
+1. Pass in the Liquid template and ask for a React component
+   1. Be sure to leave stub components for any `{% include '...' %}`s
+2. Generate unit tests for the functionality
+3. Repeat steps 1 - 2 for each include; replace the stub components with these fully-functional ones
+
+## Approved LLMs
+
+I think just GitHub Copilot provided under VA's GitHub organization.
```

Comment:
Somebody correct me if I'm wrong. I know VA employees have access to a VAGPT or some such, but that's not really useful to mention here unless we as contractors get access to it eventually.